udata_front/theme/gouvfr/assets/js/pages/PublishingForm/Step2DescribeDataset.vue

```diff
@@ -360,6 +369,26 @@ export default defineComponent({
 
     const frequenciesUrl = getFrequenciesUrl();
     const licensesUrl = getLicensesUrl();
+    const licensesGroups = license_groups_options.map(([name, values]) => ({
+      name,
+      values
+    }));
+    const formatSpatialZones = (data) => {
+      const suggestions = data.map(item => {
+      const matchingGranularity = props.granularities.find(granularity => granularity.id === item.level);
```

Comment:
I think matchingGranularity does not show on my local setup?
![image](https://github.com/etalab/udata-front/assets/28541902/e6e0ac79-f21b-49fc-8cc2-ca8bf2fe48d8)

Reply:
Hmm looks like matchingGranularity is defined, but the description is not showing?
Should we add `:showDescription="true"`?

Reply:
Oh thank you ! I did forgot to update that component after the new version of select-a11y. You can check now with the change applied